README.md

```diff
@@ -271,7 +300,7 @@ In particular:
 
 * **Terragrunt configuration**: When using file paths directly in your Terragrunt configuration (`terraform.tfvars`),
   such as in an `extra_arguments` block, you can't use hard-coded absolute file paths, or it won't work on your
-  teammates' computers. Therefore, you should use a relative file path with the `get_tfvars_dir()` helper:
+  teammates' computers. Therefore, you should utilize `get_tfvars_dir()` to use a relative file path:
```

Comment:
It may not be obvious to readers that `get_tfvars_dir()` is a Terragrunt built-in function. Consider re-wording to clarify.